languages/haskell/app/Main.hs

```diff
@@ -184,7 +187,7 @@ getTableLine = do
 
 irrelevantLine :: Parser (Maybe String)
 irrelevantLine = do
-  endOfLine <|> (notChar '|' >> skipWhile (/= '\n') >> endOfLine)
+  endOfLine <|> (notChar '|' >> skipWhile "character" (/= '\n') >> endOfLine)
```

Comment:
O_o

Reply:
"character" is the label for the individual characters in the line. Megaparsec's combinator that corresponds to `skipWhile` ([`takeWhileP`](https://hackage.haskell.org/package/megaparsec-8.0.0/docs/Text-Megaparsec.html#v:takeWhileP)) allows setting a label for what your condition corresponds to.

I am not entirely sure what that line is supposed to do though? Does it skip any line that does not start with `|`?